packages/kotti-ui/source/kotti-filter/KtFilter.vue

```diff
@@ -129,20 +137,28 @@ export default defineComponent<Kotti.Filter.InternalProps>({
 				emit('input', [...filterListValues.value, searchFilter])
 			}
 		}
+		const toggleListVisibility = () => {
+			isListVisible.value = !isListVisible.value
+			if (isListVisible.value) tippyInstanceRef.value.show()
+			else tippyInstanceRef.value.hide()
+		}
 
 		useTippy(
-			filterListTriggerRef,
+			listTriggerRef,
 			computed(() => ({
 				appendTo: () => document.body,
 				arrow: roundArrow,
-				content: filterListContentRef.value,
-				hideOnClick: true,
+				content: listContentRef.value,
+				hideOnClick: false,
```

Comment:
does it maybe default to `false`, if yes, you can just not pass this, then, no?

Reply:
It defaults to true, according to the documentation